src/gp_instance.cpp

```diff
@@ -176,21 +179,25 @@ void GPInstance::EstimateBranchLengths(double tol, size_t max_iter) {
   std::cout << "initial likelihood: " << initial_likelihood_duration.count() << "s\n";
   std::cout << "optimization: " << optimization_duration.count() << "s or "
             << optimization_duration.count() / 60 << "m\n";
+  std::ofstream branch_length_file("_output/branch_lengths.txt");
+  branch_length_file << GetEngine()->GetBranchLengths();
```

Comment:
We could come up with a `BranchLengthsToCSV` function like `SBNParametersToCSV` that also includes indexer information. Would this be useful?

Reply:
Yes, I think it would be useful.